fastify.js

```diff
@@ -218,7 +218,7 @@ function build (options) {
     /* Deal with listen (port, cb) */
     if (typeof address === 'function') {
       cb = address
-      address = undefined
+      address = '127.0.0.1'
```

Comment:
You missed the case when neither `address` and `cb` was provided (return promise case)

I think we should do something like that:
```js
    if (typeof address === 'function') {
      cb = address
      address = undefined
    }
    address = address || '127.0.0.1'
```

Reply:
Good catch. Fixed.